packages/providers/onekey-neo-provider/src/OneKeyNeoProvider.ts

```
@@ -0,0 +1,368 @@
+      return this.provider.signTransaction(params);
+    }
+
+    async switchWalletNetwork(params: ISwitchWalletNetworkParams): Promise<ISwitchWalletNetworkResponse> {
+      return this.provider.switchWalletNetwork(params);
+    }
+
+    async switchWalletAccount(): Promise<IGetAccountResponse> {
+      return this.provider.switchWalletAccount();
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)